zapisy/templates/semester_dropdown.html

```diff
@@ -0,0 +1,24 @@
+<div class="dropdown">
+    <div class="btn-group btn-block mb-3 w-100">
+        <button class="btn btn-light" disabled>
+            <strong>Semestr {{ semester }}</strong>
+        </button>
+        <button class="btn btn-light dropdown-toggle dropdown-toggle-split"
+                href="#" role="button" id="semester-dropdown" 
+                data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+            <span class="sr-only"></span>
+        </button>
+        <div class="dropdown-menu" aria-labelledby="semester-dropdown">
+            {% for semester in all_semesters %}    
```

Comment:
```suggestion
            {% for semester in all_semesters %}
```
Nic ważnego, ale jak już zauważyłem, to lepiej bez tych spacji. (Powinniśmy mieć na to jakiś linter.)